packages/flutter_test/lib/src/finders.dart

```diff
@@ -501,6 +502,26 @@ class CommonFinders {
       skipOffstage: skipOffstage,
     );
   }
+
+  /// Makes an effort to find a back button widget.
+  ///
+  /// This finder is used by [WidgetTester.pageBack].
```

Comment:
No need to say this: it might not be true in the future, and the comment would be inaccurate then.

Also, you might describe what it looks for, and why one might want to look for this specific back button (because it has to do with navigation and the `BackButton` class.